internal/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -176,7 +176,9 @@ func (c *enrollCmd) Execute(ctx context.Context, streams *cli.IOStreams) error {
 		enrollDelay,
 		*c.options,
 		c.configStore,
-		c.backoffFactory)
+		c.backoffFactory,
+		-1, // try indefinitely, let user cancel the action
```

Comment:
Let's make the `-1` a constant like `InfiniteRetries` or `InfiniteAttempts` or something.  Will help readability in the `RETRYLOOP` code in `enroll.go` as well.